src/client/serialize/daos_serialize.c

```diff
@@ -356,7 +358,8 @@ daos_cont_serialize_props(hid_t file_id, daos_prop_t *prop_query)
 			   type == DAOS_PROP_CO_EC_PDA ||
 			   type == DAOS_PROP_CO_RP_PDA ||
 			   type == DAOS_PROP_CO_GLOBAL_VERSION ||
-			   type == DAOS_PROP_CO_ALLOCED_OID) {
+			   type == DAOS_PROP_CO_ALLOCED_OID ||
+			   type == DAOS_PROP_CO_SCRUBBER_DISABLED) {
```

Comment:
This handles serializing the new property, but the new property isn't retrieved yet. You'd also want this (I haven't tested):
https://github.com/daos-stack/daos/compare/f/scrubbing_fixes...dbohning-scrubbing_fixes?expand=1

Also, FYI that this doesn't cover mpifileutils since that's a separate repo. But that's okay. We can create a ticket for that and it will be assigned to someone familiar with mpifileutils.

Reply:
Thanks Dalton. DAOS-10340 has been created to update mpifileutils once the scrubber feature branch lands to master. I'll update daos_hdlr .